client/packages/system/src/IndicatorsDemographics/api/hooks/document/useDemographicIndicators.ts

```diff
@@ -38,6 +38,12 @@ export const useDemographicIndicators = (headerData: HeaderValue[]) => {
       return;
     }
 
+    // TODO refactor hook to remove useEffect / make headerData not required so hook useable elsewhere
```

Comment:
Will you do this in this PR or create a new issue?

Reply:
I have made a new issue for this here:
https://github.com/msupply-foundation/open-msupply/issues/4148